### ISSUE TITLE
Jest collectCoverageFrom ignore paths not working

### ISSUE THREAD

**Issue by IncredibleMrTim** [risenforces/cra-alias#5]
Hey

I use code coverage, however i have found when applying ignore paths to my jest configuration, they are no longer respected.

*package.json*
```
...
	"jest": {
		"collectCoverageFrom": [
			"src/**/**/**/*.js",
			"src/**/**/**/*.jsx",
			"!src/index.js",
			"!src/setupTests.js",
			"!src/**/stories/*.js"
		]
	},
...
```

please can you take a look at this?

**Comment by risenxxx** [comment]
Hi @IncredibleMrTim. I recently decided to move from the concept of `cra-alias` to a plugin for `craco` and created [craco-alias](https://github.com/risenforces/craco-alias).

Using this package makes it possible to use the `craco` API and modify many other configs hidden by CRA (adding aliases remained as easy as it was in `cra-alias`, and new features were added).

I also think that the bug presented in this issue should not be repeated when using `craco`. For a simple transition between packages, I added [migrating from cra-alias](https://github.com/risenforces/craco-alias#migrating-from-cra-alias) section to `craco-alias` README.

I hope you will enjoy the new solution, thanks.
